scripts/jenkins-yoctobuild-build.sh

```diff
@@ -297,10 +297,12 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
                 if ! grep "COPY --from=build" Dockerfile; then
                     gulp build
                 fi
+                # GIT_REF + GIT_COMMIT for old versions, GIT_COMMIT_TAG for newer
                 docker build \
                     -t $docker_url:pr \
                     --build-arg GIT_REF=$(git describe) \
                     --build-arg GIT_COMMIT=$(git rev-parse --short HEAD) \
+                    --build-arg GIT_COMMIT_TAG="$(git describe)_$(git rev-parse --short HEAD)" \
```

Comment:
```suggestion
                    -t $docker_url:pr \
                    --build-arg GIT_COMMIT_TAG=$(git describe) \
```

Reply:
I would like to keep GIT_REF/GIT_COMMIT in case we need to build and old GUI version (remember that mender-qa is not versioned).

I have a pending clean-up PR for few things in this script, so I might remove it then.